sklearn/feature_extraction/image.py

```diff
@@ -337,22 +337,22 @@ def extract_patches_2d(image, patch_size, max_patches=None, random_state=None):
 
     Examples
     --------
-    >>> from sklearn.datasets import load_sample_image
+    >>> from sklearn.datasets import load_sample_images
```

Comment:
this change seems unrelated

Reply:
ping @CatChenal 	can you please revert these changes